iscsi/target.go

```diff
@@ -106,8 +106,7 @@ func UpdateLun(tid int, lun int, params map[string]string) error {
 		for k, v := range params {
 			paramStr += fmt.Sprintf("%s=%s,", k, v)
 		}
-		strings.TrimSuffix(paramStr, ",")
-		opts = append(opts, "--params", paramStr)
+		opts = append(opts, "--params", strings.TrimSuffix(paramStr, ","))
```

Comment:
Good fix!

Reply:
@c3y1huang need to fix this in 1.5 and 1.4.

Reply:
https://github.com/longhorn/go-iscsi-helper/pull/68